lib/Constants.d.ts

```diff
@@ -452,6 +452,8 @@ export default interface Constants {
     // Unknown 30
     STAGE_TOPIC:                                  31;
     GUILD_APPLICATION_PREMIUM_SUBSCRIPTION:       32;
+    // Unknown 33-43
```

Comment:
Yes, I know 36-39 are in #1556 already. Not in scope of this PR, so will be fixed on merge conflict when both PRs end up being merged